projects/plugins/crm/includes/ZeroBSCRM.DAL3.Helpers.php

```diff
@@ -1183,20 +1176,33 @@ function zeroBS_getOwner($objID=-1,$withDeets=true,$objType=-1,$knownOwnerID=-1)
 	return false;
 }
 
-function zeroBS_getOwnerObj($wpUserID=-1,$simpleUserData=true){
+// phpcs:ignore Squiz.Commenting.FunctionComment.Missing
```

Comment:
```suggestion
/**
 * Retrieves the owner object based on a given WP user ID.
 * 
 * This function gets the owner's data without revealing sensitive information
 * (e.g. `user_pass`).
 * 
 * @param int $wp_user_id The WordPress user ID. Default is -1.
 * 
 * @return array|bool Returns an associative array containing the 'ID' and 'OBJ' (user data object) if successful, false otherwise.
 */
```